homeassistant/components/songpal/config_flow.py

```diff
@@ -139,7 +139,7 @@ async def async_step_import(self, user_input=None):
                 name = interface_info.modelName
         except SongpalException as ex:
             _LOGGER.error("Import from yaml configuration failed: %s", ex)
-            return self.async_abort(reason="connection")
+            return self.async_abort(reason="cannot_connect")
```

Comment:
not sure if theres some magic but does the cannot_connect actually translate here? Sincd the string is defined under error when this is abort. Havent checked internal behavior.

Reply:
This is from the import step so it's not on the UI, so no need to translate.

Reply:
gotcha, i guess the log entry just uses the key based on the code im seeing.